playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/Program.cs

```diff
@@ -5,11 +5,16 @@
 
 builder.AddAzureProvisioning();
 
-var openai = builder.AddAzureOpenAI("openai")
-                    .AddDeployment(new("gpt-35-turbo", "gpt-35-turbo", "0613"));
+#pragma warning disable ASPIRE0001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+var openai = builder.AddAzureOpenAI("openai", (_, _, _, deployments) => {
+    var deployment = deployments.Single();
+    deployment.AddOutput("modelName", x => x.Name);
```

Comment:
Isn't this an input?

Reply:
We define the model name via AddDeployment ... the callback exposes it as a CDK resource, which we can then just pipe out. There are a couple of different ways to spin this using the tools we give people today but I think this is probably the easiest (unless we had deep linking connection strings - then none of this would be needed).

Reply:
I'm just asking if this isn't part of the Deployment info we pass in?

Reply:
It is, so arguably it could just be defined as a standard C# variable. We still need the bug fixed though ;)

Reply:
Make it a variable 😄

Reply:
Done.